python/lsst/ap/association/assoc_db_sqlite.py

```diff
@@ -675,6 +675,19 @@ def _store_catalog(self, source_catalog, converter, obj_ids=None, exposure=None)
         """
         values = []
 
+        if source_catalog.getSchema() != converter.schema and \
+           converter.table_name == 'dia_sources':
+            # Create aliases to appropriate flux fields if they exist.
+            schema_names = source_catalog.getSchema().getNames()
+            if 'base_PsfFlux_flux' in schema_names and \
+               'base_PsfFlux_fluxSigma' in schema_names and \
+               'psFlux' not in schema_names and \
+               'psFluxErr' not in schema_names:
+                source_catalog.getSchema().getAliasMap().set(
```

Comment:
This is hard to read. Most linters allow the extra lines of the `if` statement to be indented further than the code below them (e.g., 8 spaces rather than 4).

Also consider assigning `source_catalog.getSchema().getAliasMap()` to a variable; that would let you fit the `set` calls on one line.